### DISCUSSION TITLE
DIA-NN 1.8.2 beta 27 - feedback

### DISCUSSION THREAD

**Discussion by witszymanski** [vdemichev/DiaNN#807] [General]
Hi! 
I do not have Twitter any more (cause it sucks), but I wanted to give some short feedback of my test: 

On 24 files, the 1.8.2 version was running approx 30% longer, but identified around 5-15% more protein IDs in complex/high intensity samples, and around 50 to 80% more IDs in super low intensity samples.

![image](https://github.com/vdemichev/DiaNN/assets/15866068/4e669e67-239b-450f-b76b-3c01bd9b6601)

1.8.1 Settings:

![image](https://github.com/vdemichev/DiaNN/assets/15866068/4fcdae9d-e591-4e3b-b93f-0a3b9ffb82e8)

1.8.2 Settings:

![image](https://github.com/vdemichev/DiaNN/assets/15866068/15176962-6a91-41b3-b803-88d12e297f4d)

I will now do the correlation and other stuff analysis and post here.

**Comment by vdemichev** [comment]
Hi Witold,

Many thanks for the feedback!

Yes, the speed of analysis might go down on Thermo Orbitrap data (it's been reported before), but should go up on timsTOF.

Best,
Vadim

**Comment by witszymanski** [comment]
I am comparing these analyses in the meantime, so I will add some graphs and conclusions once in a while:

![image](https://github.com/vdemichev/DiaNN/assets/15866068/b08d7caa-d560-4297-8594-6e714711de72)


```
> nrow(object_1_only)
[1] 171
> nrow(object_2_only)
[1] 230
```

![image](https://github.com/vdemichev/DiaNN/assets/15866068/f3046aee-3099-40f6-bf8b-d671af84495b)

**Reply by vdemichev** [reply]
I really like the numbers :) What matrix is this?

**Reply by witszymanski** [reply]
Samples 1-6 are IP samples with 2 different time points of treatment (each 3 reps). 
samples 7-12 are TurboID (7-9 bait, 10-12 control). 

Samples 13-24 is the repetition of 1-12 with slightly different SP3 protocol.  

Everything measured on Orbitrap Eclipse with uPac column.

**Reply by witszymanski** [reply]
Is this what you meant by "matrix"? :)

**Reply by vdemichev** [reply]
Yes!

**Reply by MichelMoser** [reply]
Hi, this sounds very interesting. I am dealing with low intensity and low complexity data (enriched protein toxins) as well but on a timsTOF. Is there any chance of testing DIA-NN 1.8.2 as well? Thx, Michel

**Reply by vdemichev** [reply]
Yes, it's public https://twitter.com/DemichevLab/status/1705004380557340677

**Reply by MichelMoser** [reply]
Great, thank you! Any chance this will be pushed to Git Hub soon? I would love to create a docker container out of this to run on a cluster :)

**Comment by witszymanski** [comment]
So the 1.8.2 finally works now with TimsTof DIA? :D

**Reply by MichelMoser** [reply]
yes, just finished with 2 HeLa quality control runs and the stats look amazing! 
It ran in half the time of 1.8.1. 
But i will have to do some further analyses to say more... :)

**Reply by witszymanski** [reply]
I am stupid! Of course, it works on TImsTof DIA data. I was jumping thoughts a bit too fast. Reading DiaNN GitHub while reading something on FragPipe at the same time, and there, afaik, the PASEF DIA is still not implemented.
